LICENSE

```diff
@@ -7,6 +6,7 @@ See spdx.org/licenses to see what the License Identifiers used below mean.
 
 All source code (lua):
 	(C) Auke Kok <sofar@foo-projects.org>
+	(C) MultiCraft Development Team
```

Comment:
please remove this, none of the contributions are significant enough to warrant an additional copyright claim.